common/lib/xmodule/xmodule/conditional_module.py

```diff
@@ -95,27 +143,15 @@ class ConditionalModule(ConditionalFields, XModule):
         'voted': 'voted'  # poll_question attr
     }
 
-    def _get_condition(self):
-        # Get first valid condition.
-        for xml_attr, attr_name in self.conditions_map.iteritems():
-            xml_value = self.descriptor.xml_attributes.get(xml_attr)
-            if xml_value:
-                return xml_value, attr_name
-        raise Exception(
-            'Error in conditional module: no known conditional found in {!r}'.format(
-                self.descriptor.xml_attributes.keys()
-            )
-        )
-
     @lazy
     def required_modules(self):
         return [self.system.get_module(descriptor) for
                 descriptor in self.descriptor.get_required_module_descriptors()]
 
     def is_condition_satisfied(self):
-        xml_value, attr_name = self._get_condition()
+        attr_name = self.conditions_map[self.condional_attr]
```

Comment:
The specified attribute might not be in the map, since it is a user supplied value. Should this code be defensive against that situation, or is that not possible?